templates/desktop/organisations.html

```diff
@@ -600,6 +596,31 @@ <h2>Secure enterprise management with Ubuntu Pro Desktop</h2>
     </div>
   </div>
 </section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>Virtual desktops, powered by AWS</h2>
+      <p>Ubuntu Desktop on Amazon WorkSpaces offers developers an optimised Ubuntu Desktop environment to rapidly build, test and deploy code &mdash; spinning up and tearing down instances as required.</p>
+      <p>IT organisations benefit from rapid deployment, configuration and scalability for their managed Ubuntu Desktops whilst their applications and data remain secured in the AWS cloud, even when their developers are working remotely.</p>
+      <p>
+        <a href="/aws/workspaces">Find out more&nbsp;&rsaquo;</a>
```

Comment:
From an accessibility perspective, this link text needs to be more descriptive. I've left a comment for Oliver on the doc

Reply:
Oliver suggest 'Find out more about Ubuntu WorkSpaces'